packages/thirdweb/src/react/core/utils/walletIcon.ts

```diff
@@ -129,8 +129,15 @@
 /**
  * @internal Exported for tests only
  */
-export async function fetchWalletImage(props: {
-  id: WalletId;
-}) {
+export async function fetchWalletImage(props: { id: WalletId }) {
+  const { getInstalledWalletProviders } = await import(
```

Comment:
why the async import here? this will ALWAYS be hit when the function is called (not in an optional branch) so it would be more efficient to have it as part of the main bundle regardless

Reply:
because this is core, which is imported by react native and i rather not trigger this import unless i import that function if im just grabbing an icon const